eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -219,6 +219,11 @@ steps:
     configuration: "$(BuildConfiguration)"
     msbuildArguments: "/MaxCPUCount /ConsoleLoggerParameters:Verbosity=Minimal;Summary;ForceNoAlign /t:AfterBuild /p:SignPackages=true /bl:$(Build.StagingDirectory)\\binlog\\12.SignPackages.binlog"
 
+- task: NuGetToolInstaller@1
```

Comment:
Which component was installing 4.x?

Can we do 6.x instead?

Reply:
The agent pool VSEngSS-MicroBuild2022-1ES will use NuGet tool with 4.x by default.
Previously I checked our official build, it uses 5.8.0 NuGet tool for verifying: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=6297556&view=logs&j=d80b2849-30d4-52dd-74cd-d6536ba98fe0&t=df8f2bf5-5d3a-5e1a-c7c8-4bd106082de1&l=14
Sure, I will change it to 6.x.

Reply:
Changed to 6.x.